external-table/src/libchurl.c

```diff
@@ -346,14 +346,15 @@ churl_init(const char *url, CHURL_HEADERS headers)
 /* Required for resolving localhost on some docker environments that
  * had intermittent networking issues when using pxf on HAWQ
  * However, CURLOPT_RESOLVE is only available in curl versions 7.21 and above */
-#ifdef CURLOPT_RESOLVE
+#if LIBCURL_VERSION_NUM >= 0x071500
```

Comment:
if we base off the version, then it seems it has been added in 7.21.3 : https://github.com/curl/curl/commit/1b24b89cca3c06e36b69969af8edeeaca659515b

that will make: 
```suggestion
#if LIBCURL_VERSION_NUM >= 0x071503
```

Reply:
On the other hand, should we just not compare and have compiler fail if the build version is less than 7.21.3 ? Was it only the case for centos6 or even centos5 at some point ? This current patch is safe and ok for PXF5, but maybe in PXF 6 we should do just that ?

Reply:
Great catch on the version -- I assumed the comment mean 7.21.0 but I should have checked. In GPDB 5, we [vendor cURL 7.54.0](https://github.com/greenplum-db/gpdb/blob/f9cdb519790ce36c32dc8269771880e91cddfdbe/gpAux/releng/make/dependencies/ivy.xml#L27) so we should be able to safely remove this conditional altogether.